### DISCUSSION TITLE
Blue banner is inappropriate for "check not found"

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#58141] [Pull Requests] [answered]
### Select Topic Area

Bug

### Body

Steps:
1. Visit https://github.com/the-tcpdump-group/libpcap/pull/1198/checks?check_run_id=14288927389
    <img width="1700" alt="image" src="https://github.com/community/community/assets/2119212/39367402-222a-4e25-85d5-78ce64703825">

Note the _cyan_ box:
> No check run found with ID 14288927389 for this pull request.

cyan is generally used for barely important information.

When you visited this URL, the thing you were trying to retrieve wasn't available at all, that's an ERROR, ERRORS should be RED, and errors core to the content should be in the content, not in a dismissible bar at the top.

It's very confusing. Especially if someone sent you the link from before this happened and expected you to see the state for when the URL was live and referenced content therein.

**Comment by Habbie** [comment]
It's even worse in dark mode:

![image](https://github.com/community/community/assets/525838/7eadfca3-e45d-4a48-9fa1-64c5ed661002)

**Comment by beaugunderson** [comment]
I also missed this the first few times it was there and it ended up wasting some time looking for different sources of failure.

**Comment by jsoref** [comment, marked as answer]
- Refiling as https://github.com/community/community/discussions/174918 in the hope that I'll get a bot to "accept" my feedback
